opensrp-chw/src/main/java/org/smartregister/chw/activity/MalariaProfileActivity.java

```diff
@@ -385,4 +586,40 @@ public void initializeFloatingMenu() {
                 LinearLayout.LayoutParams.MATCH_PARENT);
         addContentView(baseMalariaFloatingMenu, linearLayoutParams);
     }
+
+    interface onMemberTypeLoadedListener {
+        void onMemberTypeLoaded(MemberType memberType);
+    }
+
+    private Date formatDate(String pncDate) {
+        try {
+            SimpleDateFormat df = new SimpleDateFormat("d MMM", Locale.getDefault());
+            Date date = df.parse(pncDate);
+            return date;
+        } catch (Exception e) {
+            Timber.e(e);
+        }
+        return null;
+    }
+
+    private void setPncViews(PncVisitAlertRule pncVisitAlertRule) {
+        visitDone.setVisibility(View.GONE);
+
+        if (pncVisitAlertRule.getButtonStatus().toUpperCase().equals("DUE") || pncVisitAlertRule.getButtonStatus().toUpperCase().equals("OVERDUE")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using equalsIgnoreCase() is cleaner than using toUpperCase/toLowerCase().equals().](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426626)